kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -271,4 +281,6 @@ object KubernetesApplicationOperation extends Logging {
         "mark the application state as UNKNOWN.")
       UNKNOWN
   }
+
+  def tagToKubernetesPodName(tag: String): String = s"$KUBERNETES_POD_NAME_PREFIX$tag"
```

Comment:
can we add a `-driver` suffix?

this simple way should work, I'm not sure if the user will compliant about this behavior,